tools/database/dbc/spell_effect.go

```diff
@@ -247,7 +266,13 @@ func (effect *SpellEffect) ParseStatEffect() *stats.Stats {
 
 	case effect.EffectAura == A_MOD_RATING:
 		for _, rating := range getMatchingRatingMods(effect.EffectMiscValues[0]) {
+
 			if statMod := RatingModToStat[rating]; statMod != -1 {
+				if scalesWithIlvl && effect.Coefficient != 0 && spell.Attributes[8]&0x1000 == 0 {
```

Comment:
You're doing `spell.Attributes[8]&0x1000 == 0` here when it's been `!= 0` the previous three times, is that intended?